mkdocs.yml

```diff
@@ -1,8 +1,8 @@
 site_name: Freqtrade
 nav:
-    - About: index.md
-    - Installation: installation.md
+    - Home: index.md
     - Installation Docker: docker.md
+    - Installation: installation.md
```

Comment:
I understand your intend (to have docker as the main/easiest option) and I do not mind against this, but you moved the more particular doc page (Installation Docker) to be above the more general one (Installation), I guess this should not be this way. Later, it (Installation page) can be splitted into general and quick-install pages and the order can be
* Installation (general page, prewords/prerequisites, review of available methods and workflow)
* Installation Docker
* Installation Quick-Install script (Linux)
* Manual installation (Linux)
* Installation on Windows
* Installation on RPI
or something like that. Simple moving Installation Docker to be above Installation seems wrong for me.

Reply:
Yes i know - and i did this with full intend.

everything in the installation.md file is "advanced installation". The only section that wasn't was the exchange key generation - which honestly, is wrong to have in an installation documentation in the first place - as it's not needed until you move the bot to production.

If you go to the freqtrade main doc page https://www.freqtrade.io/en/latest/ - and assume you're a new user - the first thing you'll look for is features - then "how to install" - so as user, i pick the uppermost "install" point - assumign that's the easiest (but currently it's not).
Assuming users scroll all the way to the bottom of that page to find the install link is ... optimistic at best... maybe if they're not sure and read it all - but if the user has decided to install - that's probably not what [s]he'll do.

The `docker.md` page on the other hand contains everything you need to get up and running with docker (no need to check `installation.md`), so to me, this should be at the top - as i hope it'll remove some of the install problems ta-lib sometimes causes.

I'm also not keen on splitting the installation documentation too much / further ... having too many options in the left-hand menu might help you find the correct advanced installation method - however, it'll also make it more difficult to find other options once you're installed ... (at which point you probably don't need "installation" anymore).

Reply:
> I'm also not keen on splitting the installation documentation too much / further ... having too many options in the left-hand menu might help you find the correct advanced installation method - however, it'll also make it more difficult to find other options once you're installed ... (at which point you probably don't need "installation" anymore).

That's why the installation page should be split (probably as I suggested above) and installation pages moved into a submenu, like we have for Controlling the bot -- they will be hidden when you don't need them anymore. With a small general page for installation methods at the top of the set of the pages.

Okay, let's try to see this page above the general Installation, but this is temporary...